aep/general/0137/aep.md.j2

```diff
@@ -0,0 +1,143 @@
+# Apply
+
+In REST APIs, it is customary to make a `PUT` request to a collection's URI (for
+example, `/v1/publishers/{publisher}/books`) in order to create or update a new
```

Comment:
```suggestion
example, `/v1/publishers/{publisher}/books`) in order to create or replace a new
```

PUT is replace, and we have a standard method called Update with very different semantics, so IMO "update" is confusing here.

Reply:
I actually think the semantics will end up pretty similar (with json-merge-patch behavior or replace). But yeah I guess for proto it would be different. SGTM.